### ISSUE TITLE
Very good lecture and well detailled

### ISSUE THREAD

**Issue by charafeddine46** [wlzhao22/clecture#1]
Please do you have a next part of this course. I mean , functions, pointers, linked list, tree, recusrive function......................

**Comment by wlzhao22** [comment]
The lectures about functions (Lecture 5), pointers (Lecture 8), linked lists (Lecture 8), recursive functions (Lecture 5) are included. Tree is not touched.

**Comment by ylioily** [comment]
do you have the chinese version of the slide/ pdf? thank you

**Comment by wlzhao22** [comment]
No, I do it all in English. You can translate it with AI tool by yourself.

03/09/2025 18:10:42 "ylioily" ***@***.***> wrote:

do you have the chinese version of the slide/ pdf? thank you

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you commented.Message ID: ***@***.***>

ylioily left a comment (wlzhao22/clecture#1)

do you have the chinese version of the slide/ pdf? thank you
